diff --git a/node_modules/@rneui/base/dist/Slider/Slider.js b/node_modules/@rneui/base/dist/Slider/Slider.js
index 87dab6d..570ee31 100644
--- a/node_modules/@rneui/base/dist/Slider/Slider.js
+++ b/node_modules/@rneui/base/dist/Slider/Slider.js
@@ -15,7 +15,7 @@ import { Rect } from './components/Rect';
 import { SliderThumb } from './components/SliderThumb';
 const TRACK_SIZE = 4;
 const THUMB_SIZE = 40;
-const TRACK_STYLE = Platform.select({ web: 0, default: -1 });
+const TRACK_STYLE = 0;
 const DEFAULT_ANIMATION_CONFIGS = {
     spring: {
         friction: 7,
